src/Features/Blockcore.Features.NodeHost/NodeHostSettings.cs

```diff
@@ -52,6 +52,9 @@ public class NodeHostSettings
         /// <summary>Use HTTPS or not.</summary>
         public bool UseHttps { get; set; }
 
+        /// <summary>Use title from agent</summary>
+        public string NetworkAgent { get; set; }
```

Comment:
is the name `NetworkAgent` the right field name? it translates down the line to title so I suppose perhaps `NetworkTitle` is more correct? (agent confuses with the node p2p agent)

Reply:
Actually we are using the variable NetworkAgent in both parts, at the beginning of the Swagger and as the node agent p2p. Would you like us to create a new var?